hrm-domain/hrm-service/src/contact/contact-routes-v0.ts

```diff
@@ -146,4 +147,31 @@ contactsRouter.patch(
   },
 );
 
+contactsRouter.post(
+  '/:contactId/conversationMedia',
+  validatePatchPayload,
+  canEditContact,
```

Comment:
Good for now I think? :thinking:

Reply:
I was thinking about this.. Right now the behavior is packed with the fact that a worker "can create a contact". Could exist the day where "create" and "edit" contacts have different rules, resulting in this endpoint failing when the counselor tries to "finalize the contact"? Could that lead to missing conversation medias? Just thinking out loud here.

Reply:
You are correct, those additional handlers were added in error. This endpoint should be available to anyone who can add a contact

In a follow on PR when we add the 'finalizedAt' property, it should be forbidden on finalized contacts.

In the future I can see a scenario where conversationMedia isn't added from the front end at all, but off the back of task router events etc., then this endpoint could be not callable by flex users at all, but that's for the future